server/views/eligibility-checker/cannot-trade.html

```diff
@@ -16,6 +16,17 @@ <h1 id="pageTitle" class="govuk-heading-l">{{ pageTitle }}</h1>
 
       <h2 id="heading2" class="govuk-heading-m">What you can do with this item</h2>
 
+
+      <p id="helpText3p">Based on your answers, it is illegal to:</p>
+
+      <ul id="helpTextListp" class="govuk-list govuk-list--bullet">
+        <li>sell or hire it out</li>
+        <li>offer or arrange to sell or hire it out</li>
+        <li>keep it for sale or hire</li>
+        <li>import it into the United Kingdom for sale or hire</li>
+        <li>export it from the United Kingdom for sale or hire</li>
+      </ul>
+
```

Comment:
Same as above - unit test for this element (check existence and value). Also I'm not sure about the ID here,  remove the 'p' from the end?